platform/authorized_api_guide.md

```diff
@@ -1,13 +1,13 @@
 Authorized API guide
-========================
+====================
 
 ## Introduction
 This page describes how to access Piwik PRO API which uses
 [client credentials](https://www.oauth.com/oauth2-servers/access-tokens/client-credentials/)
 OAuth grant type for obtaining user token.
 All data is sent and received as JSON and is compliant with [JSON API](http://jsonapi.org/) specification.
 
-### Obtaining token
+### Creating API Credentials
```

Comment:
Looking at our current index I think we should have instead `Platofrm -> Authorized API guide` simple `Getting started` section that will be more intuitive to find if someone want to well... get starting using our API. 

When we get rid of Analytics classic this will be authorization solution for all documented endpoints so I don't see a reason we should hide that section in Platform module. 

![image](https://user-images.githubusercontent.com/9349437/128346844-e61405c5-69df-4a57-a6b0-204ee8156287.png)

Reply:
BTW I found a golden link [regarding API Documentation & how to write such section](https://idratherbewriting.com/learnapidoc/docapis_doc_getting_started_section.html)